mirascope/gemini/calls.py

```diff
@@ -126,6 +128,8 @@ async def call_async(self, **kwargs: Any) -> GeminiCallResponse:
             generate_content_async = self.call_params.logfire(
                 generate_content_async, "gemini", response_type=GeminiCallResponse
             )  # pragma: no cover
+        else:
+            kwargs.pop("model")
```

Comment:
I wonder if a better way to do this would to be still to just pop it and then instead of if/else just re-add it to the kwargs in the if statement of logfire?

Reply:
done